lib/internal/test_runner/reporter/lcov.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const Transform = require('internal/streams/transform');
+
+// This reporter is based on the LCOV format, as described here:
+// https://ltp.sourceforge.net/coverage/lcov/geninfo.1.php
+// Excerpts from this documentation are included in the comments that make up
+// the _transform function below.
+class LcovReporter extends Transform {
+  constructor(options) {
+    super({ ...options, writableObjectMode: true, __proto__: null });
+  }
+
+  _transform(event, _encoding, callback) {
+    if (event.type === 'test:coverage') {
```

Comment:
```suggestion
    if (event.type !== 'test:coverage') {
        return callback(null);
   }
```
nitpic: spare an indentation level

Reply:
I do enjoy saving indentation. Thanks!